Sources/SwiftDocC/Model/Section/Sections/GroupedSection.swift

```diff
@@ -19,17 +19,21 @@ extension GroupedSection {
     /// This section's content split into groups by third-level headings.
     public var taskGroups: [TaskGroup] {
         // Get the child indices of any level-3 headings
-        let h3Indices = content.indices.compactMap { index -> Int? in
+        var h3Indices = content.indices.compactMap { index -> Int? in
             guard let heading = content[index] as? Heading,
                 heading.level == 3 else {
                     return nil
             }
             return index
         }
         
-        // Task groups are delimited by headings with level 3 so, if there are none, there can be no task groups, despite otherwise matching expected structure.
-        guard !h3Indices.isEmpty else {
-            return []
+        // An index to support topics that aren't organized in topic groups.
+        let phantomH3Index = Int.min
```

Comment:
👻

Reply:
Should we consider just making this `-1`? I think that would be more obvious to me that it's an intentional choice when reading the debugger instead of `-209319034398`.